packages/web-react/src/components/UNSTABLE_Header/demo/HeaderDefault.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Container } from '../../Container';
+import { Flex } from '../../Flex';
+import { ProductLogo } from '../../ProductLogo';
+import { defaultSvgLogo } from '../../ProductLogo/demo/ProductLogoDefault';
+import UNSTABLE_Header from '../UNSTABLE_Header';
+import UNSTABLE_HeaderLogo from '../UNSTABLE_HeaderLogo';
+
+const HeaderDefault = () => {
+  return (
+    <UNSTABLE_Header>
+      <Container>
+        <Flex alignmentX="left" alignmentY="center">
+          <UNSTABLE_HeaderLogo href="/">
```

Comment:
I think it's better to use `href="#"` in the demo. This link will redirect you outside the component demo if clicked.

Reply:
IMHO, this is not a good solution either. See https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md#case-i-want-navigable-links

I think the best solution will be using some section like `#example` or bind it to the top of the page or heading using an `id`.

Reply:
We are doing it like @pavelklibani suggested everywhere right now, but based on our recent experiences with links and buttons, it is not a good approach anymore :-/